pkg/metrics/ensuarance.go

```diff
@@ -26,6 +26,10 @@ const (
 	ExecutorErrorTotal    = "executor_error_total"
 	ExecutorEvictTotal    = "executor_evict_total"
 	PodResourceErrorTotal = "pod_resource_error_total"
+
+	NodeCpuCannotBeReclaimedSeconds = "node_cpu_cannot_be_reclaimed_seconds"
+	NodeResourceRecommended         = "node_resource_recommended"
+	NodeResourceRecommendedFrom     = "node_resource_recommended_from"
```

Comment:
Can it be node_resource_recommended_from rename to node_resource_recommended_vpa, it look like is the recommended for vpa